torch/csrc/distributed/c10d/ProcessGroupUCC.cpp

```diff
@@ -1,5 +1,6 @@
 #ifdef USE_C10D_UCC
 
+#include <ATen/cuda/nvrtc_stub/ATenNVRTC.h>
```

Comment:
Dumb question, why do we need NVRTC here?
According to documentation:
> NVRTC is a runtime compilation library for CUDA C++. It accepts CUDA C++ source code in character string form and creates handles that can be used to obtain the PTX. 

Do we need any runtime compilation here?

Reply:
Nope, we do not need any runtime compilation, but we do need a Driver API that is the part of `ATenNVRTC`:
https://github.com/pytorch/pytorch/blob/08460f4baeb9257a466935f56362c6ebff8cbdd5/aten/src/ATen/cuda/nvrtc_stub/ATenNVRTC.h#L53-L57